src/miral/regenerate-miral-symbols-map.py

```diff
@@ -530,13 +530,11 @@ def _report(publish, symbol):
     miral::WaylandExtensions::zwp_virtual_keyboard_manager_v1*;
     miral::WaylandExtensions::zwlr_screencopy_manager_v1*;
     typeinfo?for?miral::WaylandExtensions::EnableInfo;
-    vtable?for?miral::WaylandExtensions::EnableInfo;
-  };
-local: *;
-};
-global:'''
+    vtable?for?miral::WaylandExtensions::EnableInfo;'''
 
-END_NEW_STANZA = ''
+END_NEW_STANZA = '''  };
+local: *;
+};'''
```

Comment:
Did the `global:` go away on purpose?

Reply:
Also this doesn't seem like a change related to this PR?

Reply:
> Also this doesn't seem like a change related to this PR?

True, it fixes the changes made in #2516. (I noticed because I changed it in the earlier version of this PR.) Want it split out?